lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -130,17 +130,21 @@ class ReportUIFeatures {
 
   _setupHeaderAnimation() {
     const scoresWrapper = this._dom.find('.lh-scores-wrapper', this._document);
-    this.headerOverlap = /** @type {number} */
-      (Number(window.getComputedStyle(scoresWrapper).marginTop.replace('px', '')));
+    const computedMarginTop = scoresWrapper.style.marginTop ?
+      window.getComputedStyle(scoresWrapper).marginTop : '0px';
+    this.headerOverlap = computedMarginTop ?
+      /** @type {number} */ (Number(computedMarginTop.replace('px', ''))) : 0;
```

Comment:
I know you're copying what was here but `parseFloat(computedMarginTop)` should do the same. (unless there's some tsc thing i'm not thinking of).
   could even do `parseFloat(computedMarginTop|| 0)` to drop your ternary?